checks/raw/fuzzing_test.go

```diff
@@ -394,6 +394,116 @@ func Test_checkFuzzFunc(t *testing.T) {
 			},
 			fileContent: "import Test.Hspec",
 		},
+		{
+			name:     "Elixir QuickCheck through PropCheck",
+			want:     true,
+			fileName: []string{"Test.exs"},
+			langs: []clients.Language{
+				{
+					Name:     clients.Elixir,
+					NumLines: 50,
+				},
+			},
+			fileContent: "use PropCheck, default_opts: &PropCheck.TestHelpers.config/0",
+		},
+		{
+			name:     "Elixir QuickCheck through StreamData",
+			want:     true,
+			fileName: []string{"Test.exs"},
+			langs: []clients.Language{
+				{
+					Name:     clients.Elixir,
+					NumLines: 50,
+				},
+			},
+			fileContent: "use ExUnitProperties",
+		},
+		{
+			name:     "Elixir with no property-based testing",
+			want:     false,
+			fileName: []string{"NoPropTest.exs"},
+			wantErr:  true,
```

Comment:
we dont want `wantErr: true` here (I know this was copy/paste from Haskell, and you did the same in the Erlang PR. It wasn't caught in either of those reviews).

Also, the test was already written in a way that it will pass regardless of what wantErr is.

```diff
			name:     "Elixir with no property-based testing",
			want:     false,
			fileName: []string{"NoPropTest.exs"},
-			wantErr:  true,
```